javascript/modules/client/src/internal/client-core.ts

```diff
@@ -4,15 +4,20 @@ import { JsonRpcProvider } from "@ethersproject/providers";
 import { Contract, ContractInterface } from "@ethersproject/contracts";
 import { IClientCore } from "./interfaces/client-core";
 import { Context } from "../context";
-import { ICreateProposal, VoteOption } from "./interfaces/dao";
+import {
+  ICreateProposal,
+  IGasFeeEstimation,
+  VoteOption,
+} from "./interfaces/dao";
 import { IDAO } from "@aragon/core-contracts-ethers";
-import { BigNumberish } from "@ethersproject/bignumber";
+import { BigNumber, BigNumberish } from "@ethersproject/bignumber";
 
 export abstract class ClientCore implements IClientCore {
   private _web3Providers: JsonRpcProvider[] = [];
   private _web3Idx = -1;
   private _signer: Signer | undefined;
   private _daoFactoryAddress = "";
+  private _gasFeeReducer = 1;
```

Comment:
Same as above, if this will remain for the foreseeable future I'd use a clearer name.